### PR TITLE
Activate coverage during tests

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,8 +67,6 @@ clean:
 	rm -rf examples/l5pc/L5PC.py
 	rm -rf examples/l5pc/x86_64
 	rm -rf examples/stochkv/x86_64
-	rm -rf .coverage
```
Comment (wvangeit) [inline Makefile:70]: I'm not sure I understand this. The 'make clean' is called before all the tests, right? Will this not have the danger of merging coverage of previous runs of the tests?

Comment (wvangeit) [reply Makefile:70]: But is 'make clean' called between the unit/functional tests? Doesn't seem like it from the tox.ini file.

Comment (AurelienJaquier) [reply Makefile:70]: It is. When you look at the logs of the tests, you have the `rm -rf ...` lines from the make clean before each test.

Comment (wvangeit) [reply Makefile:70]: Also, let's move the two 'functional:' lines together at the end, so that the unit tests run before the mod files are compiled

**Review by wvangeit — APPROVED** [review]

**Comment by AurelienJaquier** [comment]
wait don't merge yet

**Comment by wvangeit** [comment]
ok

**Comment by AurelienJaquier** [comment]
Alright, I think I'm good on my side

**Pull request merged**